logging/log.go

```diff
@@ -0,0 +1,92 @@
+package logging
+
+import (
+	"time"
+
+	grpc_logrus "github.com/grpc-ecosystem/go-grpc-middleware/logging/logrus"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc/codes"
+)
+
+func New(level string) *logrus.Logger {
+	log := logrus.New()
+
+	// Default timeFormat
+	log.SetFormatter(&logrus.JSONFormatter{
+		TimestampFormat: time.RFC3339Nano,
+	})
```

Comment:
It would be helpful if there was a way to pass the logrus global logger here instead of always creating a new logger. This way we could apply these formatter and level settings globally for the process, which will take effect on all packages that use logrus, regardless of whether they accept a `logger` to their functions.